Library/Formula/norm.rb

```diff
@@ -0,0 +1,30 @@
+class Norm < Formula
+  desc "NACK-Oriented Reliable Multicast"
+  homepage "https://www.nrl.navy.mil/itd/ncs/products/norm"
+  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"
```

Comment:
If you can just check if they are considered beta or stable that'd be enough, thanks!